src/VisualStudio/Core/Def/Implementation/ProjectSystem/VisualStudioProjectTracker.cs

```diff
@@ -626,26 +623,68 @@ private void InitializeOutputPane()
             cancellationToken.ThrowIfCancellationRequested();
             OutputToOutputWindow($"Getting project information - done (took {DateTimeOffset.UtcNow - start})");
 
+            ForceLoadProjectsWhoseDesignTimeBuildFailed(projectInfos);
+
+            CreateDeferredProjects(projectInfos, cancellationToken);
+
+            CallWithTimingLog("Pushing to workspace", FinishLoad);
+        }
+
+        private void ForceLoadProjectsWhoseDesignTimeBuildFailed(IReadOnlyDictionary<string, DeferredProjectInformation> projectInfos)
+        {
+            OutputToOutputWindow($"Loading projects whose design time builds failed - start");
```

Comment:
Not necessary for this PR but right now we have so many start/end/logging stuff here it's getting hard to follow. A disposable block that does timing might be nice.